tests/test_Com.cpp

```diff
@@ -42,16 +42,15 @@ using Catch::Matchers::StartsWith;
 
 TEST_CASE("Test Com echo", "[Com]") {
     LSST::m2cellcpp::util::Log::getLog().useEnvironmentLogLvl();
-    Config::setup("UNIT_TEST");
+    Config::setup("../configs/unitTestCfg.yaml");
```

Comment:
I think maybe you want to begin to consider to add an environment variable to assign the root of module/package or something similar. Otherwise, if you execute the code in a wrong directory, the code will fail. If you have such kind of variable, the code here might become (just a pseudo code):

```
Config::setup("{path_to_package}/configs/unitTestCfg.yaml");
```